### DISCUSSION TITLE
Allow pinch to zoom for font size when viewing a file

### DISCUSSION THREAD

**Discussion by eygraber** [community/community#60558] [Mobile]
When viewing a file it is very annoying to have to scroll horizontally to see the full code if it runs past the edge of the screen. 

It would be much easier to quickly resize the text by pinching to zoom in and out. That way I can keep the text larger when it fits on the screen, and make the text smaller when I need to read something that would otherwise be off of the screen, and then quickly set it back.

**Comment by taki-on** [comment]
Thank you for sharing your thoughts and suggestions. Zooming in and out by pinch gestures is an interesting idea indeed! I will pass this idea to our design team and see how we can make the code viewer better!

**Reply by alirezaraha3** [reply]
afficial[@paje](https://github.com/alirezaraha3)

**Comment by oSumAtrIX** [comment]
I think the code view should work similar to the web version in a mobile browser where you can freely zoom and move around the page. Only scrolling vertically and horizontally makes it slow to view the code as you are restricted to do both gestures sequentially instead of in parallel.
